tests/test_web/base.py

```diff
@@ -22,6 +22,7 @@ def setUp(self):
             """
```

Comment:
You'll need to create a new wiki page explaining everything in detail about the web-app you've build. What each file does, what are all the settings etc.